src/jvmMain/kotlin/io/beatmaps/api/review.kt

```diff
@@ -467,16 +468,26 @@ fun Route.reviewRoute() {
                             return@newSuspendedTransaction ActionResponse(false, listOf("Unauthorised"))
                         }
 
-                        ReviewReply.insert {
-                            it[userId] = user.userId
-                            it[reviewId] = req.reviewId
-                            it[text] = reply.text
-                            it[createdAt] = NowExpression(createdAt)
-                            it[updatedAt] = NowExpression(updatedAt)
-                        }.let {
-                            ActionResponse(it.insertedCount > 0, listOf())
+                        try {
+                            insertedId = ReviewReply.insertAndGetId {
+                                it[userId] = user.userId
+                                it[reviewId] = req.reviewId
+                                it[text] = reply.text
+                                it[createdAt] = NowExpression(createdAt)
+                                it[updatedAt] = NowExpression(updatedAt)
+                            }.value
+                        } catch (e: Exception) {
```

Comment:
What's this try for? Probably shouldn't be catching everything...
If you're just trying to emulate the old code I wouldn't bother with it.

Reply:
Yeah, you're right. I guess i can just fetch a SQLError (or what might apply here).
Previously, success was just determined by number of entries being greater. Because I need the I'd determine failure this way.

If you find that unnecessary I can remove it as well - I'm not sure how the API behaves, when an error is thrown. Just giving a 500 I'd guess?